ddl/index.go

```diff
@@ -637,6 +642,7 @@ func (w *addIndexWorker) backfillIndexInTxn(handleRange reorgIndexTask) (nextHan
 
 		if handleOutOfRange || len(idxRecords) == 0 {
 			nextHandle = handleRange.endHandle
+			handleOutOfRange = true
```

Comment:
I'm confused why `handleOutOfRange = true` when `len(idxRecords) == 0`

Reply:
And `nextHandle` should not be `handleRange.endHandle` when `handleRange.exclude` is false